_opensearch/query-dsl/term-vs-full-text.md

```diff
@@ -0,0 +1,233 @@
+          "line_id" : 109931,
+          "play_name" : "A Winters Tale",
+          "speech_number" : 23,
+          "line_number" : "4.4.153",
+          "speaker" : "PERDITA",
+          "text_entry" : "Not like a corse; or if, not to be buried,"
+        }
+      },
+      {
+        "_index" : "shakespeare",
+        "_id" : "103117",
+        "_score" : 14.782743,
+        "_source" : {
+          "type" : "line",
+          "line_id" : 103118,
+          "play_name" : "Twelfth Night",
+          "speech_number" : 53,
+          "line_number" : "1.3.95",
+          "speaker" : "SIR ANDREW",
+          "text_entry" : "will not be seen; or if she be, its four to one"
+        }
+      }
+    ]
+  }
+}
+...
+```
+
+For a list of all full-text queries, see [Full-text queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text/index).
+
+### Example: Exact term search
+
+If you want to search for an exact term like “HAMLET” in the `speaker` field and don't need the results to be sorted by relevance scores, a term-level query is more efficient:
+
+```json
+GET shakespeare/_search
+{
+  "query": {
+    "term": {
+      "speaker": "HAMLET"
+    }
+  }
+}
+```
+
+The response contains document matches:
+
+```json
+{
+  "took" : 5,
+  "timed_out" : false,
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "skipped" : 0,
+    "failed" : 0
+  },
+  "hits" : {
+    "total" : {
+      "value" : 1582,
+      "relation" : "eq"
+    },
+    "max_score" : 4.2540946,
+    "hits" : [
+      {
+        "_index" : "shakespeare",
+        "_id" : "32700",
+        "_score" : 4.2540946,
+        "_source" : {
+          "type" : "line",
+          "line_id" : 32701,
+          "play_name" : "Hamlet",
+          "speech_number" : 9,
+          "line_number" : "1.2.66",
+          "speaker" : "HAMLET",
+          "text_entry" : "[Aside]  A little more than kin, and less than kind."
+        }
+      },
+      {
+        "_index" : "shakespeare",
+        "_id" : "32702",
+        "_score" : 4.2540946,
+        "_source" : {
+          "type" : "line",
+          "line_id" : 32703,
+          "play_name" : "Hamlet",
+          "speech_number" : 11,
+          "line_number" : "1.2.68",
+          "speaker" : "HAMLET",
+          "text_entry" : "Not so, my lord; I am too much i' the sun."
+        }
+      },
+      {
+        "_index" : "shakespeare",
+        "_id" : "32709",
+        "_score" : 4.2540946,
+        "_source" : {
+          "type" : "line",
+          "line_id" : 32710,
+          "play_name" : "Hamlet",
+          "speech_number" : 13,
+          "line_number" : "1.2.75",
+          "speaker" : "HAMLET",
+          "text_entry" : "Ay, madam, it is common."
+        }
+      }
+    ]
+  }
+}
+...
+```
+
+The term-level queries are exact matches. So, if you search for “Hamlet”, you don’t get back any matches, because “HAMLET” is a keyword field and is stored in OpenSearch literally and not in an analyzed form.
```

Comment:
```suggestion
The term-level queries are exact matches. So if you search for “Hamlet”, you don’t receive any matches because “HAMLET” is a keyword field and is stored in OpenSearch literally and not in an analyzed form.
```